src/content/en/updates/posts/2015/07/interact-with-ble-devices-on-the-web.markdown

```diff
@@ -50,7 +50,8 @@ enable the highlighted flag, restart Chrome and you should be able to
 [scan for](#scan-for-bluetooth-devices) and [connect to](#connect-to-a-bluetooth-device)
 nearby Bluetooth devices,
 [read](#read-a-bluetooth-characteristic)/[write](#write-to-a-bluetooth-characteristic)
-Bluetooth characteristics and [receive GATT Notifications](#receive-gatt-notifications).
+Bluetooth characteristics, [receive GATT Notifications](#receive-gatt-notifications) and know when [Bluetooth device gets
```

Comment:
'and know when Bluetooth device' > 'and know when a Bluetooth device'. Please fix.